### DISCUSSION TITLE
Back Buttom in Fast Restart

### DISCUSSION THREAD

**Discussion by Dantto** [ppy/osu#30679] [Ideas] [answered]
### Type

Game behaviour

### Bug description

With the new change of the fast restart screen there s no more delay THANK GOD AND WHO EVER DID IT

but they left the back buttom animation
u can think is for those who want to go to menu but,
now that theres no delay, theres no use for the buttom

by the way, it is still working closing the beatmap

Screenshot here of the "retry screen"

![osu_2024-11-15_16-25-04](https://github.com/user-attachments/assets/9673c889-c80b-4d8b-8a11-5da04d3ad569)

I think this can have 2 approaches

Option 1
get rid of it for once
PROBLEM
now the quick acces menu panel is only accesible at the first time when played a song

Option 2
make the old waiting screen appears whe the cursor is moved
thi is a solution for the problem of option 1

PROBLEM
when playing stable almost all the time will appear this "waiting screen" were the quick menu panel appears

Possible Solution
have 2 imputs for "quick retry"
tap
faster "quick retry"

tap and hold
will let the "waiting screen" appears

hope this helps

### Version

2024.1115.2

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/17782269/compressed-logs.zip)

**Comment by peppy** [comment]
It's there for a reaosn. Maybe it can be delayed globally to only appear after 200ms or something, dunno.

**Reply by frenzibyte** [reply, marked as answer]
https://github.com/ppy/osu/pull/30863
